ArtifactDeployer 1.4.6 [RN_36403] (#1471)
SkylineCommunications/dataminer-docs: release-notes/Cloud/DCP_change_log.md

The dataminer.services platform gets updated continuously. This change log can help you trace when specific features and changes have become available.

> #### 15 May 2023 - Enhancement - ArtifactDeployer 1.4.6 - General improvements [ID_36284]

Author: ```suggestion
#### 15 May 2023 - Enhancement - ArtifactDeployer 1.4.6 - General improvements [ID_36403]
```

> #### 15 May 2023 - Enhancement - ArtifactDeployer 1.4.6 - General improvements [ID_36403]

Reviewer: ```suggestion
#### 15 May 2023 - Enhancement - ArtifactDeployer 1.4.6 - Connection improvements [ID_36403]
```

> A retry mechanism has been implemented to fetch the cloud endpoint that is needed since ArtifactDeployer 1.4.5. If eventually still no cloud endpoint can be received the module will fallback and try to access dataminer.services directly. If this is not possible the module will log the failed actions.

Reviewer: ```suggestion
A retry mechanism has been implemented to fetch the cloud endpoint. If after the retries, still no cloud endpoint can be received, the module will fall back to trying to access dataminer.services directly. If this is not possible, the module will log the failed actions.
```
